src/components/nav.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+ <div class="nav_wrapper">
```

Comment:
Why are there no corresponding semantic html tags in the entire project? It lacks even as basic as nav